src/CoreWCF.Primitives/src/CoreWCF/Channels/TextMessageEncoderFactory.cs

```diff
@@ -773,4 +767,10 @@ protected override void ReturnXmlWriter(XmlDictionaryWriter writer)
             }
         }
     }
+
+    public class ContentEncoding
+    {
```

Comment:
I had a look at the usage of this and it looks like there isn't any cross package usage. Can you duplicate this in WebHttp and leave it internal here?

Reply:
This has been addressed.